src/extension.js

```diff
@@ -198,6 +236,10 @@ observeDOM(document.body, "ytd-shelf-renderer.style-scope.ytd-item-section-rende
   hideSectionByTitle("Popular today")
 })
 
+observeDOM(document.body, "ytd-topbar-logo-renderer#logo", function () {
```

Comment:
🔍 Since YT is an SPA, we have to wait until the logo container is added to the DOM and then mount the logo menu